src/ssh/sshConfig.ts

```diff
@@ -73,6 +74,46 @@ export default class SSHConfiguration {
         return new SSHConfiguration(config);
     }
 
+    static async includeLocalSSHConfig(scopeName: string, configContent: string): Promise<boolean> {
+        const render = Handlebars.compile(`## START GITPOD {{scopeName}}
+### This section is managed by Gitpod. Any manual changes will be lost.
+
+{{{configContent}}}
+
+## END GITPOD {{scopeName}}`);
+        const newContent = render({ scopeName, configContent });
+
+        const findAndReplaceScope = async (configPath: string) => {
+            try {
+                let content = '';
+                if (await exists(configPath)) {
+                    if (!(await isFile(configPath))) {
+                        return false;
+                    }
+                    content = (await fs.promises.readFile(configPath, 'utf8')).trim();
+                } else {
+                    await fs.promises.mkdir(path.dirname(configPath), { recursive: true });
+                }
+                const scopeRegex = new RegExp(`## START GITPOD ${scopeName}.*## END GITPOD ${scopeName}`, 's');
+                if (scopeRegex.test(content)) {
+                    content = content.replace(scopeRegex, newContent);
+                } else {
+                    content = `${content}\n\n${newContent}\n\n`;
+                }
+                await fs.promises.writeFile(configPath, content);
+                return true;
+            } catch (err) {
+                // ignore
+                return false;
+            }
+        };
+        const ok = await findAndReplaceScope(getSSHConfigPath());
+        if (ok) {
+            return true;
+        }
+        return await findAndReplaceScope(systemSSHConfig);
```

Comment:
We should only update user config, if the file don't exist then let's try creating it, if fails show error to user